CONTRIBUTING.md

```diff
@@ -1,21 +1,22 @@
+# Contributing
+
 # Issues
-Before opening an issue, please make sure your problem/request doesn't exist. When you open issue, please provide the following information if possible:
+Before opening an issue, please make sure your problem or request doesn't exist. When opening an issue, please provide the following information if possible:
 
 * Example code or repo (please keep it simple and minimal)
-* Steps to reproduce
-* `react-bootstrap-table2` version
+* Steps to reproduce.
+* `react-bootstrap-table2` version.
 
-Anyway, you're welcome to open an issue to ask questions or discuss about a feature request.
+Anyway, you're always welcome to open an issue to ask questions or talk about new feature request.
```

Comment:
Additionally, questions asking and new feature requests are welcomed through the issue.

Reply:
I'm not sure about the usage "questions asking"... What about:
Additionally, asking questions and requesting new features are welcomed via [issues](https://github.com/react-bootstrap-table/react-bootstrap-table2/issues)

Reply:
Actually, `questions asking` and `new feature requests` are nouns which emphasize these behavior. However, I thought both of that are easy to understand. Therefore, it's up to you :)

Reply:
Okay, I see your point here. Thank you 💯